### PR TITLE
MULE-19416: Error launching mule 4.4.0-SNAPSHOT from embedded container

### PR DESCRIPTION
**Pull request by pabloperalta**

### CODE REVIEW

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java [hunk 1]
```diff
@@ -390,6 +391,14 @@ public static void setStartupPropertiesFile(String startupPropertiesFile) {
     MuleContainer.startupPropertiesFile = startupPropertiesFile;
   }
 
+  public void setEmbeddedMode(boolean embeddedMode) {
```
Comment (elrodro83) [inline modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java:394]: javadoc

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java [hunk 2]
```diff
@@ -390,6 +391,14 @@ public static void setStartupPropertiesFile(String startupPropertiesFile) {
...
+    this.embeddedMode = embeddedMode;
+  }
+
+  public boolean isEmbeddedMode() {
```
Comment (elrodro83) [inline modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java:398]: javadoc

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreen.java [hunk 3]
```diff
@@ -46,6 +48,11 @@
 public class MuleContainerStartupSplashScreen extends SplashScreen {
 
   private Logger LOGGER = LoggerFactory.getLogger(MuleContainerStartupSplashScreen.class);
+  private boolean embeddedMode = false;
+
+  public MuleContainerStartupSplashScreen(boolean embeddedMode) {
```
Comment (elrodro83) [inline modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreen.java:53]: javadoc

File: modules/launcher/src/test/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreenEmbeddedTestCase.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.not;
+import static org.mockito.ArgumentMatchers.contains;
+import static org.mule.runtime.container.api.MuleFoldersUtil.ARTIFACT_PATCHES_FOLDER;
+import static org.mule.runtime.core.api.util.FileUtils.newFile;
+import static org.mule.tck.junit4.matcher.StringContainsIgnoringLineBreaks.containsStringIgnoringLineBreaks;
+import static org.mule.test.allure.AllureConstants.ArtifactPatchingFeature.ARTIFACT_PATCHING;
+
+@Feature(ARTIFACT_PATCHING)
+public class MuleContainerStartupSplashScreenEmbeddedTestCase
```
Comment (elrodro83) [inline modules/launcher/src/test/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreenEmbeddedTestCase.java:25]: add `@Issue annotation`

**Review by elrodro83 — CHANGES_REQUESTED** [review]

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java (hunk 1 above)
Comment (pabloperalta) [reply modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java:400]: Done!

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java (hunk 2 above)
Comment (pabloperalta) [reply modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java:407]: Done!

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreen.java (hunk 3 above)
Comment (pabloperalta) [reply modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreen.java:57]: Done!

File: modules/launcher/src/test/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreenEmbeddedTestCase.java (hunk 4 above)
Comment (pabloperalta) [reply modules/launcher/src/test/java/org/mule/runtime/module/launcher/MuleContainerStartupSplashScreenEmbeddedTestCase.java:27]: Done!

**Review by elrodro83 — APPROVED** [review]

**Pull request merged**